getting_started/step_by_step/ui_code_a_life_bar.rst

```diff
@@ -247,11 +275,19 @@ Inside the parens after the function name, add a ``player_health``
 argument. When the player emits the ``health_changed`` signal it will send
 its current ``health`` alongside it. Your code should look like:
 
-::
+.. tabs::
+ .. code-tab:: gdscript GDScript
 
     func _on_Player_health_changed(player_health):
         pass
 
+ .. code-tab:: csharp
+
+    public void OnPlayerHealthChanged(int playerHealth)
```

Comment:
I disagree, we shouldn't sacrifice inconsistency with C# just to match a screenshot that's not a C# example. The zip contains the final version of the code if they get stuck for reference.

Reply:
@neikeq Well, one way or the other (using snake_case here, or adding a note below the screenshot) it should be clear for the reader that the method displayed in the editor and this one aren't two separate things. Which is not currently the case 😄.

Reply:
Sure I'll add a comment regarding it. As for the attribute, I don't think it's worth it. Maybe we can update the warning message box to support tabs that way you can add language specific notes/warnings/info etc.... since I'm sure C# won't be the last alternative scripting language officially supported by godot.